scripts/dqlite/cmd/main.go

```diff
@@ -96,16 +97,20 @@ func main() {
 
 	fmt.Printf("Opening database at %s\n", path)
 
-	dbApp, err := app.New(path)
+	dial := client.DefaultDialFunc
+
+	dbApp, err := app.New(path, app.WithAddress("127.0.0.1:9001"))
 	if err != nil {
 		panic(err)
 	}
 
+	fmt.Printf("0x%x %s\n", dbApp.ID(), dbApp.Address())
+
 	if err := dbApp.Ready(context.Background()); err != nil {
 		panic(err)
 	}
 
-	db, err := dbApp.Open(context.Background(), *dbTypeFlag)
+	db, err := dbApp.Open(context.Background(), "tmp")
```

Comment:
This seems a bit odd. The existing flag says "controller" or "model", but you're setting it unilaterally to "tmp". Should we be changing the doc string to "controller|model|tmp" ?

Reply:
Interestingly, you'll end up with multiple files when you dump (the snapshot file and the WAL file), also I wanted to indicate that these are just temp files, don't expect anything more.

I get that it's confusing, I'll move back to the dbTypeFlag.